### ISSUE TITLE
MilkDropPanel not working at installation

### ISSUE THREAD

**Issue by bobastien** [milkdrop2077/MilkDrop3#55]
When installing the software MilkDropPanel had an error "milk3.ini not found ! please restart milkdrop"
No amount of restarting created that .ini file so i created a blank file, started MilkDropDanel and saved and now it works.
Just mentionning it here to let you know that the file did not download.

**Comment by milkdrop2077** [comment]
yep you need to start the MilkDrop 3.exe first to have the Milk3.ini file created

**Comment by bobastien** [comment]
That's the problem, even when starting the program the milk3.in file was not created, i had to create a blank file with the same name for it to work
